server/src/test/java/org/cloudfoundry/identity/uaa/oauth/UaaTokenStoreTests.java

```diff
@@ -362,6 +353,20 @@ void testCountingTheExecutedSqlDeleteStatements() throws SQLException {
         assertTrue(after.compareTo(before) < Duration.ofMinutes(5).toNanos());
         // Expect us to call the DB only once within 5 minutes. Check this when using the data source object
         verify(mockedDataSource, atMost(1)).getConnection();
+        // When
+        doReturn(Instant.now().plus(Duration.ofDays(1))).when(timeService).getCurrentInstant();
+        for (int i = 0; i < 10; i++) {
+            performExpirationClean(store);
+            verify(mockedDataSource, atMost(i + 2)).getConnection();
+        }
+    }
```

Comment:
It is hard to understand the logic here.  Shouldn't we always expect `atMost(2).getConnection()` here instead?  It seems to me that the duration will be below 5 minutes for the whole for loop iterations.

Reply:
yes was a loop before, but now simplified it. important only . If time is gone the DB cleanup should is expected again